Makefile

```diff
@@ -0,0 +1,25 @@
+prefix ?= /usr/local
+bindir ?= $(prefix)/bin
+
+.PHONY: develop
+develop:
+	swift package generate-xcodeproj
+	open SwiftInspector.xcodeproj 
+
+.PHONY: build
```

Comment:
What do you think of adding a debug target to build in debug?

Reply:
I though about adding one for building in debug, running the CLI and running tests but given those commands are

`swift build`
`swift test`
`swift run`

adding a rape to make it look like:

`make debug`
`make test`
`make run`

didn't provide a lot of value IMO, which is why I didn't add these.